plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/recommeded-methods.tsx

```diff
@@ -28,7 +24,7 @@ export default function RecommendedMethods( {
 				/>
 				<Button
 					className="woopayments-close-button"
-					onClick={ () => setIsOpen( false ) }
+					onClick={ () => {} }
```

Comment:
Given the close button will be rendered in the step content (instead of a header), we need to figure out how to do this. Passing setIsOpen isn't clean for sure, but we need a way to do it.

I still think a header will make things easier, but there is one step in the design where it's not used 😢